src/main/java/gregtech/api/capability/impl/EnergyContainerHandler.java

```diff
@@ -128,23 +128,28 @@ protected void notifyEnergyListener(boolean isInitialChange) {
 
     public boolean dischargeOrRechargeEnergyContainers(IItemHandlerModifiable itemHandler, int slotIndex) {
         ItemStack stackInSlot = itemHandler.getStackInSlot(slotIndex);
-        if (stackInSlot.isEmpty()) {
+        if (stackInSlot.isEmpty()) { // no stack to charge/discharge
             return false;
         }
         IElectricItem electricItem = stackInSlot.getCapability(GregtechCapabilities.CAPABILITY_ELECTRIC_ITEM, null);
-        if (electricItem == null || !electricItem.canProvideChargeExternally()) {
+        if (electricItem == null) { // stack in slot is not an electric item
             return false;
         }
         int machineTier = GTUtility.getTierByVoltage(Math.max(getInputVoltage(), getOutputVoltage()));
+        if (machineTier != electricItem.getTier()) { // stack in slot is not the right voltage tier to charge/discharge
```

Comment:
Instead of locking it to the exact tier, is it possible to change this to the machine tier being equal or greater than the item tier, without causing energy draw issues? It's very annoying forcing LV machines when charging LV items, etc.